islands/navigation.tsx

```diff
@@ -40,10 +41,17 @@ export const Navigation = ({ state }) => {
     return "h-5 w-5 items-center justify-center rounded-full text-red-800 bg-red-100 dark:bg-red-900 dark:text-red-300 inline-flex items-center text-sm font-medium";
   };
 
+  const getIconClass = () => {
+    const style = state.direction === "rtl"
+      ? `absolute inset-y-0 left-0 flex items-center pl-3`
+      : `absolute inset-y-0 right-0 flex items-center pr-3`;
+    return style;
+  };
```

Comment:
It's better to use `end` and `start` when dealing with positionning, padding, margin etc 
```suggestion
  const getIconClass = () => {
    return `absolute inset-y-0 end-0 flex items-center pe-3`
  };
```

https://tailwindcss.com/docs/top-right-bottom-left